src/Microsoft.IdentityModel.Tokens/X509EncryptingCredentials.cs

```diff
@@ -11,6 +11,8 @@ namespace Microsoft.IdentityModel.Tokens
     /// </summary>
     public class X509EncryptingCredentials : EncryptingCredentials
     {
+        internal const string _useShortNameForRsaOaepKey = "Switch.Microsoft.IdentityModel.UseShortNameForRsaOaepKey";
```

Comment:
```suggestion
        internal const string UseShortNameForRsaOaepKey = "Switch.Microsoft.IdentityModel.UseShortNameForRsaOaepKey";
```

Reply:
This follows the naming convention of other switches used in Wilson. Thanks Peter!

Reply:
I'd argue they are incorrect (I think there's only one other switch here?) but we are inconsistent with naming throughout...

Usually (per Microsoft guidelines) constants are in PascalCase.